src/graphql/queries/maps.ts

```diff
@@ -0,0 +1,30 @@
+import gql from 'graphql-tag';
+import { Event } from './events';
+
+export interface LocationPin {
+  id: string;
+  eventId: Event;
+  label: string;
+  latitude: number;
+  longitude: number;
+  address: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+export const GET_PINS_BY_EVENT_ID = gql`
+  query pinsForEvent($eventId: ID!) {
+    pinsForEvent(eventId: $eventId) {
+      id
+      eventId {
```

Comment:
Just want to confirm that this is okay given that event queries return more info.

Reply:
I believe this should be fine as we don't need any more information. All we really need is the ID for caching purposes. cc @petorsong what do you think?

Reply:
i'm ok with this as long as it doesn't have the potential to introduce caching issues in the future

this does bring up a good question on how we should structure our graphql queries (or which one is more of a graphql-ism)

we can either
1. have big/wide queries that fetch a lot of stuff from the backend (e.g. `events`, `patients`) and only do filtering for what we need in the frontend; this may force us to fetch a lot of data in cases where we don't necessarily need it
OR
2. have queries for very specific use cases (e.g. `ccpsForEvent`, `patientsForCcp`); this seems like it won't leverage apollo's caching capabilities to the fullest and may require more network requests in the long run

we're kind of in the middle of both options right now, lemme know if either of you have any opinions for either option